### PR TITLE
Update dep nsqjs to compatible with node 10

### PR DESCRIPTION
**Pull request by wwayne**

### CODE REVIEW

File: test/producer.test.js [hunk 1]
```diff
@@ -279,12 +279,17 @@ describe('producer', () => {
             ]).on('close', (code) => {
               startNsqd((e) => {
                 setTimeout(() => { //wait to reconnection
-                  p.produce(TOPIC, 'message after reconnect', (error) => {
-                    expect(error).to.not.exist;
-                    spawn('docker-compose', [
-                      `--file=${composeFile}`, 'stop', 'nsqd3'
-                    ]);
-                    done();
+                  p.connect((e) => {
```
Comment (xavierchow) [inline test/producer.test.js]: why here connect again? the Produce is supposed to be able to reconnect automatically, no?

Comment (wwayne) [reply test/producer.test.js]: no, will get the error like no connection, even after waiting 15s

Comment (xavierchow) [reply test/producer.test.js]: does it mean the auto-reconnect feature doesn't work on node 10? need to figure it out instead of bypassing the test code.

Comment (wwayne) [reply test/producer.test.js]: I haven't checked, I thought you missed connect in the testing... will check

**Comment by xavierchow** [comment]
@wwayne don't bump to 1.1 in this PR,
it should be handled by another PR, you may notice there is an obvious decrease of coverage which indicates some path is not covered anymore.

**Comment by wwayne** [comment]
you're right, just removed my last commit

**Pull request merged**